arangod/Transaction/Manager.cpp

```diff
@@ -784,6 +784,25 @@ futures::Future<Result> Manager::ensureManagedTrx(
   co_return res;
 }
 
+namespace {
+futures::Future<futures::Unit> sleepUsingSchedulerIfAvailable(
+    const char* reason, std::chrono::milliseconds duration) {
```

Comment:
nit: we can as well use std::string_view here. we are only calling it with C string literals as of now, which are null-terminated.
```suggestion
    std::string_view reason, std::chrono::milliseconds duration) {
```